composer.json

```diff
@@ -19,17 +19,17 @@
         "php": ">=7.0.8",
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "drupal-composer/drupal-scaffold": "^2.5",
+        "drupal/core-composer-scaffold": "^8.8.0",
+        "drupal/core": "^8.8.0",
```

Comment:
The reason I originally made this ^8.7 instead of ^8.8 was to emphasize that older versions of Drupal also work. These two values will both evaluate in the same way, of course -- 8.8.0 is selected today, 8.9.0 will be selected when it is available, etc.